frontend/src/components/UserDetail/UserDetail.tsx

```diff
@@ -133,6 +149,13 @@ const UserDetail = ({
             ) : (
               <RiderModal existingRider={rider} isRiderWeb={isRider} />
             )}
+              <Switch>
+                <Route path="/riders">
+                  <button className = {styles.back_div}onClick={studentDelete}>
+                    <img className={styles.trashIcon} alt="trash" src={red_trash} />
+                  </button>
+                </Route>
+              </Switch>
```

Comment:
shouldn't do this, you can put the button in the parens of `rider && !isRider ? ( ... )` on line 129. everything there will only display on admin web